llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -2721,6 +2721,12 @@ class TargetLoweringBase {
                                      Type *Ty, unsigned AddrSpace,
                                      Instruction *I = nullptr) const;
 
+  /// Return the perfered common base offset.
+  virtual int64_t getPreferBaseOffset(int64_t MinOffset,
```

Comment:
Maybe call this something like getPreferredLargeGEPBaseOffset?

Reply:
apply your comment, thanks